content/en/docs/installation/compatibility.md

```diff
@@ -29,5 +29,20 @@ plane nodes in the [GKE
 docs](https://cloud.google.com/kubernetes-engine/docs/how-to/private-clusters#add_firewall_rules).
 
 
+## EKS
```

Comment:
Can we make it `AWS EBS` or `Amazon EKS`? all these KSes get confusing over time

Reply:
Sure, replaces it with `AWS EKS`